src/components/common/form/FormAccessibilityProvider.tsx

```
@@ -0,0 +1,406 @@
+import React, { useEffect } from "react";
+
+interface FormAccessibilityProviderProps {
+  formRef: React.RefObject<any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)